src/record/record-transition.js

```diff
@@ -131,7 +132,8 @@ RecordTransition.prototype.add = function (socketWrapper, version, message) {
     }
 
     try {
-      this._applyConfig(update, message)
+      const config = utils.getRecordConfig(message)
```

Comment:
utils is for generic stuff, this would make more sense to just be a static method on this class